app/Resources/assets/sass/styles.scss

```diff
@@ -29,6 +29,10 @@ header .navbar-default {
     height:60px;
 }
 
+.bottomPadd8{
+    padding-bottom: 8px !important;
```

Comment:
we shouldn't use `!important`, is this strongly necessary? did you test nothing else is affected with this rule?